corehq/apps/users/management/commands/gdpr_scrub_user_from_forms.py

```diff
@@ -0,0 +1,50 @@
+from __future__ import print_function
+from __future__ import absolute_import
+from __future__ import unicode_literals
+from django.core.management.base import BaseCommand
+from corehq.apps.users.models import CommCareUser
+from corehq.form_processor.interfaces.dbaccessors import FormAccessors
+from io import StringIO
+from lxml import etree
+import sys
+import six
+import logging
+
+
+logger = logging.getLogger(__name__)
+
+
+class Command(BaseCommand):
+    help = "Scrubs the username from all forms associated with the given user"
+
+    def add_arguments(self, parser):
+        parser.add_argument('username')
+        parser.add_argument('domain')
+
+    def handle(self, username, domain, **options):
+        this_form_accessor = FormAccessors(domain=domain)
+        user = CommCareUser.get_by_username(username)
+        if not user:
+            logger.info("User {} not found.".format(username))
+            sys.exit()
```

Comment:
should say `sys.exit(1)`, which indicates an error condition

Reply:
https://github.com/dimagi/commcare-hq/pull/19979/commits/4b45c9117c40d43ca06f3d5dfeaae87f0d68cc38